### ISSUE TITLE
Overdue task email not being sent

### ISSUE THREAD

**Issue by xboxhaxorz** [chiefonboarding/ChiefOnboarding#469]
I have some tasks for my test account that sent about a wk ago and i havent gotten any overdue reminders, i am getting other emails

When i go to the dashboard for that account it shows the tasks as overdue

All 3 checkboxes are set in the settings in regards to emails

I have a 2nd test account made and that account isnt getting the overdue email either

**Comment by GDay** [comment]
You are right. Good catch. Not sure why no one else has complained about this yet. ChiefOnboarding went through a massive rewrite back in the day and it looks like this functionality never made it back in the code. 
Will add it to my list to put this back.

**Comment by xboxhaxorz** [comment]
It looks as though the login credentials email is also not being sent, i made 3 users and neither of them got it, when i do it from the settings, it does send it, its just not automatic

Tasks are being sent fine though

I still have all 3 email settings enabled in the settings page

**Comment by GDay** [comment]
Those emails are only send if the user is already on their first day (or after) when being created or if they get there over time (a background task is ran in that case that takes care of it).

**Comment by xboxhaxorz** [comment]
So i created the user on the 13th, with their active date being on the 14th, the user currently is now active as they got the email that says you have X todo and X resource, they did not get the welcome email with credentials

This is a fresh test user with a new email
